### ISSUE TITLE
Use salesforce to send receipt emails

### ISSUE THREAD

**Issue by ScottDowne** [MozillaFoundation/Advocacy#395]
Migrated from https://github.com/mozilla/donate.mozilla.org/issues/935

Right now it's sawmill.

**Comment by valianttry** [comment]
@ScottDowne if this is related to #612 can we close this one?

**Comment by ScottDowne** [comment]
@valianttry nah,, #612 is far too meta, we can use this one as a single built ticket. It's blocked on Salesforce, though.

@stephaniemcv where should we plop this into as a milestone?

**Comment by stephaniemcv** [comment]
Mid q1?

**Comment by WillatMozFdn** [comment]
how about early Q3 2018?  Better late than never.  It's on our list for the upcoming LCM sprint.
